### ISSUE TITLE
Data not converted to cuda in VAE examples(VAE/main.py)

### ISSUE THREAD

**Issue by ksharsha** [pytorch/examples#110]
In the VAE example, although the model is converted to cuda if cuda support is present, the data and the loss functions are not converted t0 cuda, resulting in an error when training on GPU.

**Comment by soumith** [comment]
I've just ran the VAE example with both v0.1.10 and against master. Both of them seem to run without error both on CUDA and on CPU.

What is your pytorch version?

**Comment by ksharsha** [comment]
I seem to have used the earlier version of this file which fixed the above error.
